Examples/README.md

```diff
@@ -0,0 +1,59 @@
+# Examples
+
```

Comment:
To give some visibility to the examples in the repo, I think it would be good to include them in this README. Something like

```
## Examples

- Command line tool to add one to every integer literal in a source file (AddOneToIntegerLiterals.swift)
- Code-generate a simple source file using SwiftSyntaxBuilder (SourceFile.swift)
```

And adding links to the example source files would also be good IMO.